web/templates/navbar.hbs

```diff
@@ -1,73 +1,76 @@
 <div class="header">
-    <nav class="header-main" id="top_navbar">
-        <div class="column-left">
-            <a class="brand no-style" href="#">
-                <img id="realm-logo" src="" alt="" class="nav-logo no-drag"/>
-            </a>
-        </div>
-        <div class="column-middle" id="navbar-middle">
-            <div class="column-middle-inner">
-                <div id="streamlist-toggle" {{#if embedded}}class="hide-streamlist-toggle-visibility"{{/if}} title="{{t 'Stream list' }} (q)">
-                    <a id="streamlist-toggle-button" role="button" tabindex="0"><i class="fa fa-reorder" aria-hidden="true"></i>
-                        <span id="streamlist-toggle-unreadcount">0</span>
-                    </a>
-                </div>
-                <div class="top-navbar-border top-navbar-container">
-                    <div id="message_view_header" class="notdisplayed">
+    {{!-- remove `top_navbar_full_width` wrapper div once the scrollbar is on the `html` element, and then move the styles on `header` to `navbar-container --}}
```

Comment:
Which scrollbar are we expecting to move to the `html` element? I'm not sure what future plan this is referring to.

Reply:
And do we really need an extra div rather than applying styles to either the `header` or `header-main` element? I am fine with it if we do, but I also don't understand what this div does, since it appears to have no styles that would make it a different size from the elements around it?

Reply:
Maybe you can help me write a clearer comment, but here's a video of what I mean. The scrollbar is currently on `app`, so the navbar needs to have height 0. `header` has bottom padding that needs to be the app background color, and `top_navbar` is centered on big screens with auto margin on its sides so it can't fill the full width with the header color.

https://user-images.githubusercontent.com/5634097/231659007-890d5f39-5a55-4f9e-81b6-dbca9a9f9d4f.mp4




red is `header`. I added a wrapper around `top_navbar` to make its color full width

<img width="782" alt="image" src="https://user-images.githubusercontent.com/5634097/231658912-508a70aa-6f9f-476c-b3f4-3c244b3de037.png">